client/components/LicenseSelect/LicenseSelect.tsx

```diff
@@ -33,25 +31,14 @@ const LicenseSelect = (props: Props) => {
 	const { children, onSelect, persistSelections, pubData, updateLocalData } = props;
 	const [isPersisting, setIsPersisting] = useState(false);
 	const { communityData } = usePageContext();
-
-	const currentLicense = getLicenseBySlug(pubData.licenseSlug);
-	// @ts-expect-error ts-migrate(2532) FIXME: Object is possibly 'undefined'.
-	const primaryCollectionPub = pubData.collectionPubs.find((cp) => cp.isPrimary);
-	const collectionPubDate = primaryCollectionPub
-		? primaryCollectionPub.collection.metadata.copyrightYear ||
-		  primaryCollectionPub.collection.metadata.date ||
-		  primaryCollectionPub.collection.metadata.publicationDate
-		: null;
-	const pubCopyrightDate =
```

Comment:
This fallthrough will never occur because `dateFormat(null)` returns a string representing `Date.now()`. As a result we never actually show the correct Pub publication date.